test/test-isProbablyReaderable.js

```diff
@@ -6,18 +6,106 @@ var expect = chai.expect;
+  });
+
+  it("should declare small and large documents as readerable when lower minContentLength", function () {
+    var options = { minContentLength: 120, minScore: 0 };
+    expect(isProbablyReaderable(verySmallDoc, options), "very small doc").to.be.false;
+    expect(isProbablyReaderable(smallDoc, options), "small doc").to.be.true;
+    expect(isProbablyReaderable(largeDoc, options), "large doc").to.be.true;
+    expect(isProbablyReaderable(veryLargeDoc, options), "very large doc").to.be.true;
+  });
+
+  it("should only declare largest document as readerable when higher minContentLength", function () {
+    var options = { minContentLength: 200, minScore: 0 };
+    expect(isProbablyReaderable(verySmallDoc, options), "very small doc").to.be.false;
+    expect(isProbablyReaderable(smallDoc, options), "small doc").to.be.false;
+    expect(isProbablyReaderable(largeDoc, options), "large doc").to.be.false;
+    expect(isProbablyReaderable(veryLargeDoc, options), "very large doc").to.be.true;
+  });
+
+  it("should declare small and large documents as readerable when lower minScore", function () {
+    var options = { minContentLength: 0, minScore: 4 };
+    expect(isProbablyReaderable(verySmallDoc, options), "very small doc").to.be.false; // score: ~3.3
+    expect(isProbablyReaderable(smallDoc, options), "small doc").to.be.true; // score: ~11.4
+    expect(isProbablyReaderable(largeDoc, options), "large doc").to.be.true; // score: ~11.9
+    expect(isProbablyReaderable(veryLargeDoc, options), "very large doc").to.be.true; // score: ~24.4
+  });
+
+  it("should declare large documents as readerable when higher minScore", function () {
+    var options = { minContentLength: 0, minScore: 11.5 };
+    expect(isProbablyReaderable(verySmallDoc, options), "very small doc").to.be.false; // score: ~3.3
+    expect(isProbablyReaderable(smallDoc, options), "small doc").to.be.false; // score: ~11.4
+    expect(isProbablyReaderable(largeDoc, options), "large doc").to.be.true; // score: ~11.9
+    expect(isProbablyReaderable(veryLargeDoc, options), "very large doc").to.be.true; // score: ~24.4
+  });
+
+  it("should use node visibility checker provided as option - not visible", function () {
+    var called = false;
+    var options = {
+      visibilityChecker() {
+        called = true;
+        return false;
+      }
+    };
+    expect(isProbablyReaderable(veryLargeDoc, options)).to.be.false;
+    expect(called).to.be.true;
+  });
+
+  it("should use node visibility checker provided as option - visible", function () {
+    var called = false;
+    var options = {
+      visibilityChecker() {
+        called = true;
+        return true;
+      }
+    };
+    expect(isProbablyReaderable(veryLargeDoc, options)).to.be.true;
+    expect(called).to.be.true;
+  });
+
+  it("should use node visibility checker provided as parameter - not visible", function () {
+    var called = false;
+    var visibilityChecker = () => {
+      called = true; return false;
```

Comment:
Nit: split this into two lines (same below) ?

Reply:
```suggestion
      called = true;
      return false;
```